CONTRIBUTING.md

```diff
@@ -144,7 +167,7 @@ Make their life easier.
 Archivematica uses the Python [PEP8](https://www.python.org/dev/peps/pep-0008/) community style guidelines.
 Newly-written code should conform to PEP-8 style.
 PEP8 is a daunting document, but there are very good linters available that you can run to check style in your code.
-The Archivematica development uses [Sublime Text](sublimetext.com/) with the [SublimeLinter-flake8](https://github.com/SublimeLinter/SublimeLinter-flake8) plugin, which automatically scans your code and displays any style notes inline.
+The Archivematica development team uses [Sublime Text](sublimetext.com/) with the [SublimeLinter-flake8](https://github.com/SublimeLinter/SublimeLinter-flake8) plugin, which automatically scans your code and displays any style notes inline.
```

Comment:
This is not true of all Archivematica committers, I don't think.  sublime is non-free software, I would prefer not to advocate for its use specifically.  Maybe this could just be changed to strongly suggest using a linter, and to suggest using flake8 instead of just pylint?